package.json

```
@@ -8,7 +8,7 @@
     "start": "node production/index.js",
     "dev": "nodemon --exec babel-node src/index.js",
     "coverage": "nyc report --reporter=text-lcov | coveralls",
-    "test": "nyc --reporter=html --reporter=text mocha  --require  @babel/register  src/tests/*.js --exit",
+    "test": "export NODE_ENV=test && sequelize db:migrate:undo:all  && sequelize db:migrate && nyc --reporter=html --reporter=text mocha  --require  @babel/register --require @babel/polyfill  src/tests/*.js --exit",
```

Here remove the `export NODE_ENV=test && sequelize db:migrate:undo:all  && sequelize db:migrate &&`
 add:
```suggestion
    "pretest": "export NODE_ENV=test && sequelize db:migrate:undo:all  && sequelize db:migrate",
    "test": "sequelize db:migrate && nyc --reporter=html --reporter=text mocha  --require  @babel/register --require @babel/polyfill  src/tests/*.js --exit",
```